targets/opsis_base.py

```diff
@@ -43,63 +43,105 @@ def __init__(self, platform, clk_freq):
         self.clk8x_wr_strb = Signal()
         self.clk8x_rd_strb = Signal()
 
+        # Input 100MHz clock
         f0 = 100*1000000
         clk100 = platform.request("clk100")
         clk100a = Signal()
+        # Input 100MHz clock (buffered)
         self.specials += Instance("IBUFG", i_I=clk100, o_O=clk100a)
         clk100b = Signal()
-        self.specials += Instance("BUFIO2", p_DIVIDE=1,
-                                  p_DIVIDE_BYPASS="TRUE", p_I_INVERT="FALSE",
-                                  i_I=clk100a, o_DIVCLK=clk100b)
+        self.specials += Instance(
+            "BUFIO2", p_DIVIDE=1,
+            p_DIVIDE_BYPASS="TRUE", p_I_INVERT="FALSE",
+            i_I=clk100a, o_DIVCLK=clk100b)
+
         f = Fraction(int(clk_freq), int(f0))
         n, m = f.denominator, f.numerator
         assert f0/n*m == clk_freq
         p = 8
         pll_lckd = Signal()
         pll_fb = Signal()
-        pll = Signal(6)
-        self.specials.pll = Instance("PLL_ADV", p_SIM_DEVICE="SPARTAN6",
-                                     p_BANDWIDTH="OPTIMIZED", p_COMPENSATION="INTERNAL",
-                                     p_REF_JITTER=.01, p_CLK_FEEDBACK="CLKFBOUT",
-                                     i_DADDR=0, i_DCLK=0, i_DEN=0, i_DI=0, i_DWE=0, i_RST=0, i_REL=0,
-                                     p_DIVCLK_DIVIDE=1, p_CLKFBOUT_MULT=m*p//n, p_CLKFBOUT_PHASE=0.,
-                                     i_CLKIN1=clk100b, i_CLKIN2=0, i_CLKINSEL=1,
-                                     p_CLKIN1_PERIOD=1e9/f0, p_CLKIN2_PERIOD=0.,
-                                     i_CLKFBIN=pll_fb, o_CLKFBOUT=pll_fb, o_LOCKED=pll_lckd,
-                                     o_CLKOUT0=pll[0], p_CLKOUT0_DUTY_CYCLE=.5,
-                                     o_CLKOUT1=pll[1], p_CLKOUT1_DUTY_CYCLE=.5,
-                                     o_CLKOUT2=pll[2], p_CLKOUT2_DUTY_CYCLE=.5,
-                                     o_CLKOUT3=pll[3], p_CLKOUT3_DUTY_CYCLE=.5,
-                                     o_CLKOUT4=pll[4], p_CLKOUT4_DUTY_CYCLE=.5,
-                                     o_CLKOUT5=pll[5], p_CLKOUT5_DUTY_CYCLE=.5,
-                                     p_CLKOUT0_PHASE=0., p_CLKOUT0_DIVIDE=p//8,  # sdram wr rd
-                                     p_CLKOUT1_PHASE=0., p_CLKOUT1_DIVIDE=6,
-                                     p_CLKOUT2_PHASE=230., p_CLKOUT2_DIVIDE=p//4,  # sdram dqs adr ctrl
-                                     p_CLKOUT3_PHASE=210., p_CLKOUT3_DIVIDE=p//4,  # off-chip ddr
-                                     p_CLKOUT4_PHASE=0., p_CLKOUT4_DIVIDE=p//2,
-                                     p_CLKOUT5_PHASE=0., p_CLKOUT5_DIVIDE=p//1,  # sys
+
+        unbuffered_sdram_full = Signal()
```

Comment:
We should get the names changed in the MiSoC targets too. The `pll[5]` ends up with signals in the generated files as `pll_5` which makes things like error messages and timing reports much harder to read. 

For example, you currently get;
```
+-------------------------------+-------------+-------------+-------------+-------------+-------------+-------------+-------------+
|TSclk100                       |     10.000ns|      5.340ns|      9.980ns|            0|            0|            0|      4876528|
| TS_hdmi2usbsoc_clk100b        |     10.000ns|      3.334ns|      9.980ns|            0|            0|            0|      4847676|
|  TS_hdmi2usbsoc_pll_3_        |      5.000ns|      1.730ns|          N/A|            0|            0|            0|            0|
|  TS_hdmi2usbsoc_pll_1_        |     15.000ns|     14.951ns|          N/A|            0|            0|       133078|            0|
|  TS_hdmi2usbsoc_pll_5_        |     20.000ns|     18.846ns|          N/A|            0|            0|      4711465|            0|
|  TS_hdmi2usbsoc_pll_0_        |      2.500ns|          N/A|          N/A|            0|            0|            0|            0|
|  TS_hdmi2usbsoc_pll_4_        |     10.000ns|      9.457ns|          N/A|            0|            0|         2755|            0|
|  TS_hdmi2usbsoc_pll_2_        |      5.000ns|      4.990ns|          N/A|            0|            0|          378|            0|
```

Afterwards;
```
|TSclk100                       |     10.000ns|      0.925ns|      9.958ns|            0|            0|            0|      4202487|
| TS_hdmi2usbsoc_clk100b        |     10.000ns|      3.334ns|      9.958ns|            0|            0|            0|      4202487|
|  TS_hdmi2usbsoc_unbuffered_sdr|      5.000ns|      1.730ns|          N/A|            0|            0|            0|            0|
|  am_half_b                    |             |             |             |             |             |             |             |
|  TS_hdmi2usbsoc_unbuffered_enc|     15.000ns|     14.893ns|          N/A|            0|            0|       130795|            0|
|  oder                         |             |             |             |             |             |             |             |
|  TS_hdmi2usbsoc_unbuffered_sys|     20.000ns|     19.552ns|      7.861ns|            0|            0|      4041421|        27160|
|  TS_hdmi2usbsoc_unbuffered_sdr|      2.500ns|          N/A|          N/A|            0|            0|            0|            0|
|  am_full                      |             |             |             |             |             |             |             |
|  TS_hdmi2usbsoc_unbuffered_sys|     10.000ns|      9.119ns|          N/A|            0|            0|         2754|            0|
|  2x                           |             |             |             |             |             |             |             |
|  TS_hdmi2usbsoc_unbuffered_sdr|      5.000ns|      4.979ns|          N/A|            0|            0|          357|            0|
|  am_half_a                    |             |             |             |             |             |             |             |
```

It might be better to have a little shorter names though....